CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ _Mar 16, 2023_
 We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
 
 - 🚀 Fire `onChange` when filling a partial date (#8082) @flaviendelangle
+- 🎁 Support format like `1st` / `2nd` (#8188) @flaviendelangle
```

Comment:
I wonder if we should say "in the pickers" for those lines
Because without context, some of them are very hard to understand. Is it for the data grid ? Is it for the pickers ?

Or to group the highlights.

I think it would be nice to improve that point in the future.

Reply:
For this line, maybe we can just say `Support date formats like...`, that raises most of the uncertainty.

Reply:
What do you think about this?
```suggestion
- 🎁 Support date format like `1st` (`do`) (#8188) @flaviendelangle
```

Reply:
That would be fine for me
I do think a broader discussion will be needed, especially after the 1st release of the chart. But let's not do it in a release PR :laughing: